agent/xds/server.go

```diff
@@ -170,7 +170,7 @@ func (s *Server) StreamAggregatedResources(stream ADSStream) error {
 
 	err := s.process(stream, reqCh)
 	if err != nil {
-		s.Logger.Debug("Error handling ADS stream", "error", err)
+		s.Logger.Error("Error handling ADS stream", "error", err)
```

Comment:
While debugging a failing integration test I noticed these errors were set to debug. 

I suspect there is a reason for this, but it caught me by surprise. I missed the error the first few times.  Is Debug the right level here? Error seems more appropriate, otherwise these errors would likely be missed in production deployments.

Reply:
This seems like the right level to me.